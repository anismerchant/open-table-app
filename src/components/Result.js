import React from 'react';
import { Card } from 'antd';

const { Meta } = Card;

export default (props) => (     
  <Card 
    hoverable
    style={{ width: 240, margin: 15 }}
    cover={<img alt="restaurant" src={props.image_url}  />}
  >
    <Meta title={props.name} description={props.address} />
    <p>{props.city}, {props.state} {props.postal_code}</p>
    <p>${props.price}</p>
  </Card>
);

