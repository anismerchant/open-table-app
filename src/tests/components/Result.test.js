import React from 'react';
import { shallow } from 'enzyme';
import Result from '../../components/Result';

test('It should correctly render Result.', () => {
  const wrapper = shallow(<Result />);
  expect(wrapper).toMatchSnapshot();
});
