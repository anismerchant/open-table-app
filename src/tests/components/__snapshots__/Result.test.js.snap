// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should correctly render Result. 1`] = `
<Card
  cover={
    <img
      alt="restaurant"
    />
  }
  hoverable={true}
  style={
    Object {
      "margin": 15,
      "width": 240,
    }
  }
>
  <Meta />
  <p>
    , 
     
  </p>
  <p>
    $
  </p>
</Card>
`;
